core/src/main/java/brooklyn/catalog/CatalogLoadMode.java

```diff
@@ -20,8 +20,11 @@
 
 import org.slf4j.LoggerFactory;
 
+import brooklyn.catalog.internal.CatalogInitialization;
 import brooklyn.entity.rebind.persister.PersistMode;
 
+/** @deprecated since 0.7.0 replaced by {@link CatalogInitialization} */
```

Comment:
I think this class has been added _and_ deprecated in 0.7.0. Can we just get rid of it?